yarn.lock

```diff
@@ -4487,6 +4487,15 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@calcom/zohocalendar@workspace:packages/app-store/zohocalendar":
```

Comment:
👍
This is the only acceptable change in yarn.lock when we add an app. If that app adds a some dependency, then there can be other things.